interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncountermanagerController.php

```diff
@@ -140,17 +144,7 @@ public function indexAction()
 
     public function buildCCDAHtml($content)
     {
-        $xml = simplexml_load_string($content);
-        $xsl = new DOMDocument();
-        // cda.xsl is self contained with bootstrap and jquery.
-        // cda-web.xsl is used when referencing styles from internet.
-        $xsl->load(__DIR__ . '/../../../../../public/xsl/cda.xsl');
-        $proc = new XSLTProcessor();
-        $proc->importStyleSheet($xsl); // attach the xsl rules
-        $outputFile = sys_get_temp_dir() . '/out_' . time() . '.html';
-        $proc->transformToURI($xml, $outputFile);
-
-        return file_get_contents($outputFile);
+        return $this->getEncountermanagerTable()->getCcdaAsHTML($content);
```

Comment:
No problem I can see. As long as the same indexaction from api route are followed which this doesn't bother, my service is good.